tidb-cloud/data-service-postman-integration.md

```diff
@@ -0,0 +1,93 @@
+
+In TiDB Cloud [Data Service](https://tidbcloud.com/console/data-service), you can easily import your Data App to Postman and leverage Postman's extensive tools to enhance your API development experience.
+
+This document describes how to import your Data App to Postman and how to run your Data App in Postman.
+
+## Before you begin
+
+Before importing a Data App to Postman, make sure that you have the following:
+
+- A [Postman](https://www.postman.com/) account.
+- A [Postman desktop app](https://www.postman.com/downloads) (Optional. You can use the Postman webpage version without downloading the app.)
+- A [Data App](/tidb-cloud/data-service-manage-data-app.md) with at least one [endpoint](/tidb-cloud/data-service-manage-endpoint.md). Only endpoints that meet the following requirements can be imported to Postman:
+
+    - The target cluster is selected.
+    - The endpoint path and request method are configured.
+    - The SQL statements are written.
+
+- An [API key](/tidb-cloud/data-service-api-key.md#create-an-api-key) for the Data App.
+
+## Step 1. Import your Data App to Postman
+
+To import your Data App to Postman, take the following steps:
+
+1. In the [TiDB Cloud console](https://tidbcloud.com), navigate to the [**Data Service**](https://tidbcloud.com/console/data-service) page of your project.
+2. In the left pane, click the name of your target Data App to view its details.
+3. In the upper-right corner of the page, click **Run in Postman**. A dialog for import instructions is displayed.
+
+    > **Note:**
+    >
+    > - If a Data App does not have any endpoint with the target cluster, path, request method, and SQL statements configured, **Run in Postman** is unclickable for the Data App.
+    > - For a Chat2Query Data App, **Run in Postman** is unavailable.
+
+4. Follow the steps provided in the dialog for the Data App import:
+
+    1. Depending on the Postman version you want to use, click **Run in Postman for Web** or **Run in Postman Desktop** to open your Postman workspaces, and then select your target workspace in Postman.
+
+        - If you have not logged into Postman, follow the on-screen instructions to log into Postman first.
+        - If you clicked **Run in Postman Desktop**, follow the on-screen instructions to open the Postman desktop app.
+
+    2. On the page of your target workspace in Postman, click **Import** in the left navigation menu.
+    3. Copy the Data App URL in the TiDB Cloud dialog, and then paste the URL to Postman for the import.
+
+5. After you paste the URL, Postman imports the Data App automatically as a new [collection](https://learning.postman.com/docs/collections/collections-overview). The name of the collection is in the `TiDB Data Service - <Your App Name>` format.
+
+    In the collection, the deployed endpoints are grouped in the **Deployed** folder and the undeployed endpoints are grouped in the **Draft** folder.
```

Comment:
In the collection, the deployed endpoints are grouped under the **Deployed** folder and the un-deployed endpoints under the **Draft** folder.